DOM_creation/lesson1/main.js

```diff
@@ -1,3 +1,4 @@
+// Lesson 1
 // Get list-item id element
```

Comment:
IMO
I think it would be better to delete comments that are obvious from the code.